### ISSUE TITLE
train.py epochs default

### ISSUE THREAD

**Issue by DrExplosive** [AI4EPS/PhaseNet#73]
The epochs default number is set to 100 in train.py, but the descriptions says 'number of epochs (default: 10)' on line 21. Which number is recommended when retraining? Thanks.

![code](https://user-images.githubusercontent.com/46575643/222463496-9453ad36-4eee-4a80-94ed-4dd0407cdaa0.png)

**Comment by zhuwq0** [comment]
Hi, the default is 100. You would need to monitor your loss function to determine a good epoch number. The training code is pretty old now. You can try https://github.com/seisbench/seisbench developed by @yetinam.

**Comment by DrExplosive** [comment]
Hey, thanks for this.
